app.js

```diff
@@ -36,4 +36,7 @@ app.use(function (err, req, res, next) {
   res.json({ success: false, err, msg: err.message });
 });
 
+//Serving static files
+app.use('/static', express.static('public'))
```

Comment:
This line is a duplicate of the changes in line 20

Reply:
Removed static file serving in app.js